lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -72,9 +72,9 @@ def colorized_formatted_backtrace(colorizer=::RSpec::Core::Formatters::ConsoleCo
           end
         end
 
-        def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes)
+        def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes, skip=0)
           lines = fully_formatted_lines(failure_number, colorizer)
-          lines.join("\n") << "\n"
+          lines.drop(skip).join("\n") << "\n"
```

Comment:
I agree. The "drop 2 first lines" thing is more specific to the way html formatter displays the exception than to the "fully_formatted" method.